javascript/src/seedsource/components/Layers.jsx

```diff
@@ -2,32 +2,98 @@ import React from 'react'
 
 
 class Layers extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            displayResults: true,
+            displayVariables: true,
+            displaySeedZones: false
+        }
+    }
+
     render() {
         let { onToggleLayer, layers } = this.props
-        let layerList = (urlSegment) => layers.filter(layer => layer.urlTemplate.includes(urlSegment))
-            .map(layer => {
-                return (
-                    <div key={layer.name}>
-                        <input
-                            type="checkbox"
-                            value={layer.name}
-                            onChange={() => onToggleLayer(layer.name)}
-                            checked={layer.displayed}
-                            disabled={(layer.urlTemplate === "seedZone") ? true : false}
-                        />
-                        {layer.name}
-                    </div>
-                )
+
+        let layerList = (urlSegment, sort=false) => layers.filter(layer => layer.urlTemplate.includes(urlSegment))
+            .sort((a,b) => {
+                if (sort) {
+                    let x = a.name.toLowerCase()
+                    let y = b.name.toLowerCase()
+                    if (a.displayed === true && b.displayed === false) {
+                        return -1
+                    } else if (a.displayed === false && b.displayed === true) {
```

Comment:
Why not use `localeCompare` here, too?

Reply:
Actually I will just remove this since seedzones are already sorted, unless you think a sort option would be useful in the future?

Reply:
That's fine.